lib/rucio/daemons/reaper/reaper2.py

```diff
@@ -594,31 +616,33 @@ def stop(signum=None, frame=None):
     GRACEFUL_STOP.set()
 
 
-def run(threads=1, chunk_size=100, once=False, greedy=False, rses=None, scheme=None, exclude_rses=None, include_rses=None, vos=None, delay_seconds=0, sleep_time=60):
+def run(threads=1, chunk_size=100, once=False, greedy=False, rses=None, scheme=None, exclude_rses=None, include_rses=None, vos=None, delay_seconds=0, sleep_time=60, auto_exclude_threshold=100, auto_exclude_timeout=600):
     """
     Starts up the reaper threads.
 
-    :param threads:            The total number of workers.
-    :param chunk_size:         The size of chunk for deletion.
-    :param threads_per_worker: Total number of threads created by each worker.
-    :param once:               If True, only runs one iteration of the main loop.
-    :param greedy:             If True, delete right away replicas with tombstone.
-    :param rses:               List of RSEs the reaper should work against.
-                               If empty, it considers all RSEs.
-    :param scheme:             Force the reaper to use a particular protocol/scheme, e.g., mock.
-    :param exclude_rses:       RSE expression to exclude RSEs from the Reaper.
-    :param include_rses:       RSE expression to include RSEs.
-    :param vos:                VOs on which to look for RSEs. Only used in multi-VO mode.
-                               If None, we either use all VOs if run from "def",
-                               or the current VO otherwise.
-    :param delay_seconds:      The delay to query replicas in BEING_DELETED state.
-    :param sleep_time:         Time between two cycles.
+    :param threads:                The total number of workers.
+    :param chunk_size:             The size of chunk for deletion.
+    :param noaccess_threshold:     Number of service unavailable exceptions after which the RSE gets temporarily excluded.
```

Comment:
`noaccess_threshold` doesn't exist as parameter

Reply:
thanks, I missed that one.